python/ct/crypto/cert.py

```diff
@@ -844,6 +844,21 @@ def ocsp_responders(self):
         return [a[x509_ext.ACCESS_LOCATION] for a in aia
                 if a[x509_ext.ACCESS_METHOD] == oid.ID_AD_OCSP]
 
+    def embedded_sct_list(self):
+        """Get the encoded list of embedded timestamps
+
+        Returns:
+            bytes representing a TLS encoded SignedCertificateTimestampList
+
+        Raises:
+            CertificateError: corrupt extension, or multiple extension values.
+        """
+        esl = self._get_decoded_extension_value(oid.CT_EMBEDDED_SCT_LIST)
```

Comment:
Naming: esl is not a comprehensible variable name. embedded_sct_list is ok, signed_certificate_timestamp_list is even better because it maps directly to a data structure in RFC6962 so it's clear what it holds.